src/Raven.Server/Config/Categories/ReplicationConfiguration.cs

```diff
@@ -6,7 +6,14 @@ namespace Raven.Server.Config.Categories
 {
     public class ReplicationConfiguration : ConfigurationCategory
     {
-        [DefaultValue(600)]
+		[Description("Threshold under which a connection is considered active")]
+		[DefaultValue(30)]
+		[TimeUnit(TimeUnit.Seconds)]
+		[ConfigurationEntry("Raven/Replication/ActiveConnectionTimeoutInSec")]
+		[ConfigurationEntry("Raven/Replication/ActiveConnectionTimeout")]
```

Comment:
this concept was introduced by me and @arek. This way we will know what was the old key and it will be easier to write the migration document for configuration values at the end. New keys should only have one ConfigurationEntry.